devicesetup/src/main/java/io/particle/android/sdk/devicesetup/model/DeviceCustomization.java

```diff
@@ -0,0 +1,592 @@
+    private int troubleshootingUri = R.string.troubleshooting_uri;
+
+    //FIXME: The customization.xml file says: support static HTML for all these
+    private int termsOfServiceHtmlFile = R.string.terms_of_service_html_file;
+    private int privacyPOlicyHtmlFile = R.string.privacy_policy_html_file;
+    private int getForgotPasswordHtmlFile = R.string.forgot_password_html_file;
+    private int troubleshootingHtmlFile = R.string.troubleshooting_html_file;
+
+    private int showSignUpPageFinePrint = R.bool.show_sign_up_page_fine_print;
+
+    private int pageBackgroundColor = R.color.page_background_color;
+    private int formFieldBackgroundColor = R.color.form_field_background_color;
+    private int normalTextColor = R.color.normal_text_color;
+    private int linkTextColor = R.color.link_text_color;
+    private int linkTextBackground = R.color.link_text_bg;
+    private int errorTextColor = R.color.error_text_color;
+
+    private int elementBackgroundColor = R.color.element_background_color;
+    private int elementBackgroundColorDark = R.color.element_background_color_dark;
+    private int elementTextColor = R.color.element_text_color;
+    private int elementTextDisabledColor = R.color.element_text_disabled_color;
+
+    /*
+    The font names below are the paths to the font you want to use, relative to the assets
+        folder.
+
+         e.g.: if you've put comic_sans.ttf in src/main/assets/fonts folder,
+          then below, then the string value setted here should be like
+
+            <string name="my_font">fonts/comic_sans.ttf</string>
+     */
+
+    private int normalTextFontName = R.string.normal_text_font_name;
+    private int boldTextFontName = R.string.bold_text_font_name;
+    private int italicTextFontName = R.string.italic_text_font_name;
+    private int headerTextFontName = R.string.header_text_font_name;
+
+    private int organization = R.bool.organization;
+    private int organizationSlug = R.string.organization_slug;
+    private int productSlug = R.string.product_slug;
+
+    //FIXME: The customization.xml file says: apparently these values aren't customizable on the iOS app?
+    private int indicatorLight = R.string.indicator_light;
+    private int powerOnAction = R.string.power_on_action;
+
+
+    //FIXME: The customization.xml file says: figure out how these should relate to the colors defined above
+    private int primaryColor = R.color.element_background_color;
+    private int primaryColorDark = R.color.primary_color_dark;
+    private int accentColor = R.color.element_background_color;
+    private int systemBarBackground = R.color.system_bar_bg;
+
+    private int edittextFromElementGap = R.dimen.edittext_form_element_gap;
+
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+    //Getters and Setters:
+
+    public int getBrandName() {
+        return brandName;
+    }
+
+    public void setBrandName(int brandName) {
```

Comment:
This might be nice with a fluent interface (i.e.: make the setters all return the current `DeviceCustomization` instance).

Thoughts?